app/javascript/components/public_health/query/AdvancedFilter.js

```diff
@@ -273,6 +276,26 @@ class AdvancedFilter extends React.Component {
     });
   };
 
+  // Reset modal when cancelled
+  cancel = () => {
+    // check if there was a filter applied
+    const appliedFilterId = localStorage.getItem(`SaraFilter`);
+    const appliedFilter = this.state.savedFilters.find(filter => {
+      return filter.id === parseInt(appliedFilterId);
+    });
+
+    // update state accordingly
+    const applied = appliedFilterId;
```

Comment:
Not a huge thing, but with how screwy JS is with types, im typically not a fan on implicit casting. Maybe use something like `!!` or `_.isNil(appliedFilterId)`?

Reply:
done